queue/queue.js

```diff
@@ -13,13 +13,27 @@ var queueApp = angular.module("queueApp", ["ngMaterial", "ngMessages"])
 		QueueService.getAppointments(year + "-" + month + "-" + day).then(function(data) {
 			if(data == null) {
 				console.log('server error');
-				// TODO
 			} else if(data.length > 0) {
 				$scope.appointments = data.map((appointment) => {
 					// This map converts the MySQL Datatime into a Javascript Date object
 					var t = appointment.scheduledTime.split(/[- :]/);
 					appointment.scheduledTime = new Date(Date.UTC(t[0], t[1]-1, t[2], t[3], t[4], t[5]));
-
+					appointment.checkedIn = false;
+					if (appointment.timeIn != null) {
+						appointment.checkedIn = true;
```

Comment:
Couldn't you simplify these conditional statements to be of the form:
`appointment.checkedIn = appointment.timeIn != null;` 
And similarly for the ones below?